### ISSUE TITLE
Rails 7 - Generated models and my own custom changes aren't applying.

### ISSUE THREAD

**Issue by alaarab** [lml/commontator#200]
I run rake commontator:copy:models and make changes to the validation rules inside model in app/models/commontator/comment.rb but they don't apply, like the documentation says it should.

**Comment by alaarab** [comment]
I am trying to remove the validation for uniqueness on comment body. If there was a way to do this with a config flag, I also wouldnt need to copy out the model and change it, although it would be nice to still be able to update the models.

**Comment by Dantemss** [comment]
If you are in dev mode, did you remember to restart the server after copying the file?
It's possible something changed in Rails 7 and broke this feature though, since I know they changed how they load files.

**Comment by alaarab** [comment]
yeah I definitely tried that and it didn't work. I also tried generating a brand new project on latest rails and Ruby and still no luck unfortunately. Looks like it is an issue with rails 7.

I ended up forking and modifying the model myself for the short term until a better solution is available.

**Comment by zebleck** [comment]
got the same issue, for comments_controller

**Comment by musik** [comment]
try add this line to your application.rb
`Commontator::Engine.config.autoload_once_paths = [] `

**Comment by alaarab** [comment]
@musik  it seems the first option worked for me.

**Comment by BBVishalkumar** [comment]
> try add this line to your application.rb `Commontator::Engine.config.autoload_once_paths = [] ` or `Commontator::Engine.config.autoload_once_paths.delete_if{|s| s.match? /models/}`

This worked for me 
Thank you
